agent/provider/tests/rule_cli.rs

```diff
@@ -172,8 +172,13 @@ fn rule_set_should_edit_certificate_rules(rule: &str, mode: &str) {
         .success();
 
     let result = list_rules_command(data_dir.path());
+    let mode_actual = result["outbound"][rule]
+        .as_object()
+        .and_then(|obj| obj.iter().find(|(id, _cert)| id.starts_with(&cert_id)))
+        .map(|(_id, value)| &value["mode"]);
 
-    assert_eq!(&result["outbound"][rule][&cert_id]["mode"], mode);
+    assert!(mode_actual.is_some());
+    assert_eq!(mode_actual.unwrap(), mode);
```

Comment:
Two asserts here doesn't make sense. Second one will Oslo fail when `mode_actual` will be none so I propose either:
1. Leave only second one as is
2. Leave second one as so: `assert_eq!(mode_actual, mode);` and `unwrap` is before assert when creating `let mode_actual`